wire/internal/wire/parse.go

```diff
@@ -773,50 +777,57 @@ func processInterfaceValue(fset *token.FileSet, info *types.Info, call *ast.Call
 // isInjector checks whether a given function declaration is an
 // injector template, returning the wire.Build call. It returns nil if
 // the function is not an injector template.
-func isInjector(info *types.Info, fn *ast.FuncDecl) *ast.CallExpr {
+func isInjector(info *types.Info, fn *ast.FuncDecl) (*ast.CallExpr, error) {
```

Comment:
Perhaps a better name for this function is `findInjectorBuild` or some such.

Reply:
Done.